packages/libllvm/build.sh

```diff
@@ -88,9 +88,13 @@ termux_step_pre_configure() {
 	cp $TERMUX_PKG_BUILDER_DIR/nl_types.h openmp/runtime/src/android
 	cp $TERMUX_PKG_BUILDER_DIR/nltypes_stubs.cpp openmp/runtime/src/android
 
-	# Add unknown vendor, otherwise it screws with the default LLVM triple
-	# detection.
+	# Change first hyphen to add unknown vendor,
+	# otherwise it screws with the default LLVM triple detection.
+	# Remove default Termux Android API level in LLVM triple and
+	# handle detection in clang.
 	export LLVM_DEFAULT_TARGET_TRIPLE=${CCTERMUX_HOST_PLATFORM/-/-unknown-}
+	LLVM_DEFAULT_TARGET_TRIPLE=${LLVM_DEFAULT_TARGET_TRIPLE/androideabi${TERMUX_PKG_API_LEVEL}/androideabi}
+	LLVM_DEFAULT_TARGET_TRIPLE=${LLVM_DEFAULT_TARGET_TRIPLE/android${TERMUX_PKG_API_LEVEL}/android}
```

Comment:
I don't think we should change this default, as it is also used to compile LLVM for Android.

Reply:
This changes default target in `clang -v` and defines in `echo | clang -E -dM -` which omitted the API level.

For building LLVM will rely on `<android/api-level.h>` instead which is not touched at all and kept at API level 24.

Reply:
> For building LLVM, we already put --target=arch-unknown-linux-android24 everywhere in build script

Right, but [we're also setting `LLVM_HOST_TRIPLE` to this just below](https://github.com/termux/termux-packages/blob/e20c8974b4acc2c6ffb65b235595dd61d1d403bc/packages/libllvm/build.sh#L110), so if our `CFLAGS/CXXFLAGS` are not pulled in properly somewhere, we'd be building that LLVM file with a target triple without an API level.

I would prefer to keep this the same when building clang for Android, and only override it when running clang _on_ Android.

Anyway, not asking you to change this now, as I'm leaning against this whole pull atm.

Reply:
When API level is not specified, build will fallback to use `<android/api-level.h>`